release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -212,6 +212,12 @@ For more information, see xref:../networking/hardware_networks/switching-bf2-nic
 
 For more information, see xref:../networking/hardware_networks/configuring-hardware-offloading.adoc#supported_devices_configuring-hardware-offloading[Supported devices].
 
+[id="ocp-4-13-expand-cluster-network-ip-address-range"]
+==== Expand cluster network IP address range
+
+// TODO - fix xref w/ working link
+The cluster network can be expanded to support the addition of nodes to the cluster. For example, if you deployed a cluster and specified `10.128.0.0/19` as the cluster network range and a host prefix of `23`, you are limited to 16 nodes. You can expand that to 510 nodes by changing the CIDR mask on a cluster to `/14`. For more information, see ref:../networking/configuring-cluster-network-range.adoc#configuring-cluster-network-range[Configuring the cluster network range].
```

Comment:
```suggestion
The cluster network can be expanded to support the addition of nodes to the cluster. For example, if you deployed a cluster and specified `10.128.0.0/19` as the cluster network range and a host prefix of `23`, you are limited to 16 nodes. You can expand that to 510 nodes by changing the CIDR mask on a cluster to `/14`. For more information, see xref:../networking/configuring-cluster-network-range.adoc#configuring-cluster-network-range[Configuring the cluster network range].
```

Reply:
@GroceryBoyJr, thanks, I left it that way so it wouldn't fail the preview build; fixed now.